tx.go

```diff
@@ -29,6 +29,19 @@ type Tx struct {
 	pages          map[pgid]*page
 	stats          TxStats
 	commitHandlers []func()
+
+	// WriteToFlag specifies the flag for WriteTo method.
+	// WriteTo opens the database file with the specified
+	// flag to copy the data.
+	//
+	// By default, the flag is set to o_direct to avoid cache
+	// trashing for large on disk workload.
+	// For in-memory workload, clear o_direct can avoid unnecessary
+	// disk IO.
+	//
+	// If open file fails with the given flag, WriteTo will fail
+	// back to use empty flag.
+	WriteToFlag int
```

Comment:
Can you change the name to WriteFlag? The interface is WriterTo but this flag would apply to other write methods if they were added later.